cardea/cardea.py

```diff
@@ -0,0 +1,221 @@
+import json
+from inspect import isclass
+
+import featuretools as ft
+import pandas as pd
+
+import cardea
+from cardea.data_loader import EntitySetLoader
+from cardea.featurization import Featurization
+from cardea.modeling import Modeler
+from cardea.problem_definition import (
+    DiagnosisPrediction, LengthOfStay, MissedAppointmentProblemDefinition, MortalityPrediction,
+    ProlongedLengthOfStay, Readmission)
+
+
+class Cardea():
+    """"An interface class that ties the end-to-end system together.
```

Comment:
there is an extra `"`

![image](https://user-images.githubusercontent.com/4410771/53917467-9262c600-4032-11e9-9a77-20602917de92.png)